translations/monero-core_fr.ts

```diff
@@ -1286,7 +1286,7 @@ Description : </translation>
     <message>
         <location filename="../main.qml" line="731"/>
         <source>Please wait...</source>
-        <translation>Veuillez patienter...</translation>
+        <translation>Veuillez patienter…</translation>
```

Comment:
OK. Was just curious if it was changed by you or automatically by your text editor. :)